modules/ROOT/pages/using-api-alerts.adoc

```diff
@@ -2,46 +2,42 @@
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
-:keywords: alerts,api, api alerts
+:keywords: alerts, api, api alerts
 
-An API alert is an alarm that flags one of the following problems:
+Anypoint Platform alerts you about request behavior, performance issues, actions related to client application contracts, and runtime events. The platform supports three types of alerts: API alerts, contracts alerts, and runtime alerts. API alerts and contracts alerts are managed by Anypoint API Manager (API Manager). Runtime alerts are managed by Anypoint Runtime Manager (Runtime Manager).
 
-* The API request violates a policy.
-* Requests received by the API exceed a given number within a period of time.
-* The API returns a specified HTTP error code.
-* The API response time exceeds a certain amount.
+== API Alerts
+ 
+Add and configure API alerts that flag unusual API behavior. Examples include:
 
-xref:runtime-manager::alerts-on-runtime-manager.adoc[Alerts] in Runtime Manager differ from API alerts. For example, Runtime Manager flags CPU usage, secure data gateway up and down, and deployment success or failure. API alerts relate to an API request or response.
+* The response time of an API exceeds 60 seconds.
+* The number of API requests in 30 seconds is greater than 1000. 
 
-You can add one of the following types of alerts to an API:
+See xref:add-api-alert-task.adoc[Adding an API Alert].
 
-* Policy violation
-+
-An infraction of one or more policies that govern the API occurs.
-+
-* Request Count
-+
-Users request access to the API more times than allowed in a specified period of time.
-+
-* Response Code
+API Manager enables you to add the following API alerts to an API instance:
+
+* *Policy Violation*
 +
-The API returns one of these HTTP codes upon receiving a request: 400, 401, 403, 404, 408, 500, 502, or 503
+An infraction of one or more policies governing the API occurs.
+
+* *Request Count*
 +
-* Response Time
+Users request access to the API more times than allowed in a specified interval of time.
 +
-The response time of the API exceeds a specified period.
-
-A request count alert condition is similar to a policy alert that uses the rate limiting policy. Valid entries for the time interval option for a request count are in the range 1 - 999999999. These time intervals specified are implemented as sliding windows, not absolute minute boundaries, to prevent problems caused by data bursts.
+Specify time interval values within the range 1 - 999999999. To prevent data bursts, intervals are implemented as sliding windows instead of as absolute minute boundaries. See https://www.techopedia.com/definition/869/sliding-window[Sliding Windows].
```

Comment:
GLOBALLY for ranges, to omit any confusion, replace the "-" symbol with "to" or "through" to clarify whether the second value is included.
In fact..."between" and "within the range" do not explicitly include the first number, either. It might be best to use wording like "values from x to[or through] y."

Reply:
Fixed